torch/onnx/symbolic_opset9.py

```diff
@@ -1970,3 +1966,33 @@ def gelu(g, self):
     erf = g.op('Erf', div(g, self, torch.tensor(_sqrt2)))
     erf_plusone = add(g, erf, g.op('Constant', value_t=torch.tensor(1, dtype=torch.float)))
     return mul(g, mul(g, self, erf_plusone), g.op('Constant', value_t=torch.tensor(0.5, dtype=torch.float)))
+
+
+@parse_args('v', 'i', 'v', 'v', 'f', 'i')
+def group_norm(g, input, num_groups, weight, bias, eps, cudnn_enabled):
+    input_sizes = input.type().sizes()
+    shape = [0, num_groups, -1]
+    input_reshaped = g.op('Reshape', input, g.op('Constant', value_t=torch.LongTensor(shape)))
+
+    # Due to shape difference. we need to apply weight and bias after
```

Comment:
Is instance norm can handle all the cases? If dim C is not divisible by group, are we still able to use instancenorm to represent groupnorm?

Reply:
What I understanf from the spec (https://pytorch.org/docs/stable/_modules/torch/nn/modules/normalization.html#GroupNorm)
is that C is always divisible by group

Reply:
If this is not the case, torch complains, for example:
RuntimeError: Expected number of channels in input to be divisible by num_groups, but got input of shape [20, 6, 10, 10] and num_groups=4